www/directives/missions/submit.html

```diff
@@ -1,7 +1,76 @@
-<div class="form-group" ng-controller="MissionSubmitCtrl">
-	<form ng-submit="submit()">
-		<label for="source">Source Code:</label>
-		<textarea class="form-control" rows="15" id="source" ng-model="source"></textarea>
-		<input type="submit" id="mission-submit" value="submit" class="btn btn-primary btn-lg" />
-	</form>
+<div>
+	<div class='col-xs-12 col-md-6' ng-init='initCodeMirror()'>
+		<div class='panel panel-default'>
+			<form ng-submit='submit()'>
+				<textarea id='source' ng-model="source"></textarea>
+				<div class='panel-footer text-right'>
+					<input type='submit' value='Try code' class='btn btn-success' />
+				</div>
+			</form>
+		</div>
+	</div>
+	<div class='col-xs-12 col-md-6' ng-if='result'>
+		<!-- Mission success -->
+		<div class='panel panel-success' ng-if='result.status == "success"'>
+			<div class='panel-body text-center'>
+				<h3 class='text-success'>
+					Congratulations, you accomplished this mission!
```

Comment:
I'm currently working on a full css/display/text rewrite, next version will be better